packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,85 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface Node {
```

Comment:
```
{ /* Node */
                          "node":
                            [ [ /^([a-zA-Z0-9][a-zA-Z0-9])*$/,
                                { /* Value */
                                  "value": $Context_hash }
                                || { /* Node */
                                     "node": $Context_hash } ] ... ] }
```
Seems to be an array of array where the first element is a string `/^([a-zA-Z0-9][a-zA-Z0-9])*$/` and second element is value or node